homeassistant/components/shelly/cover.py

```diff
@@ -0,0 +1,100 @@
+"""Cover for Shelly."""
+from aioshelly import Block
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    SUPPORT_CLOSE,
+    SUPPORT_OPEN,
+    SUPPORT_SET_POSITION,
+    SUPPORT_STOP,
+    CoverEntity,
+)
+from homeassistant.core import callback
+
+from . import ShellyBlockEntity, ShellyDeviceWrapper
```

Comment:
`ShellyBlockEntity` is moved to `.entity` in #39747

Reply:
Done